src/components/KeyboardShortcutsModal.js

```diff
@@ -53,16 +53,16 @@ class KeyboardShortcutsModal extends React.Component {
         this.unsubscribeCloseEnterModal = KeyboardShortcut.subscribe(closeShortcutEnterModalConfig.shortcutKey, () => {
             ModalActions.close();
             KeyboardShortcutsActions.hideKeyboardShortcutModal();
-        }, closeShortcutEnterModalConfig.descriptionKey, closeShortcutEnterModalConfig.modifiers, true, () => !this.props.isShortcutsModalOpen);
+        }, closeShortcutEnterModalConfig.descriptionKey, closeShortcutEnterModalConfig.modifiers, true, () => !this.props.isShortcutsModalOpen, 0, true, ['TEXTAREA']);
```

Comment:
```suggestion
        }, closeShortcutEnterModalConfig.descriptionKey, closeShortcutEnterModalConfig.modifiers, false, () => !this.props.isShortcutsModalOpen);
```

Hmm this option seems to also solve the problem. We already had a prop to exclude inputs.

Reply:
@Pujan92 I think using `captureOnInputs` sounds a bit better, right?

Reply:
Yes @aldo-expensify, I am not sure then why excludedNodes was added but will check on it.